Wrappers/Python/cil/processors/CofR_FBP.py

```diff
@@ -0,0 +1,260 @@
+# -*- coding: utf-8 -*-
+#   This work is part of the Core Imaging Library (CIL) developed by CCPi 
+#   (Collaborative Computational Project in Tomographic Imaging), with 
+#   substantial contributions by UKRI-STFC and University of Manchester.
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#   http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+from cil.framework import Processor, AcquisitionData
+from cil.processors.Binner import Binner
+import matplotlib.pyplot as plt
+import scipy
+import numpy as np
+import inspect
+import logging
+import math
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+class CofR_FBP(Processor):
+
+    r'''CofR_FBP processor maximises the sharpness of a reconstructed slice.
+
+    For use on data-sets that can be reconstructed with FBP.
```

Comment:
You never explain what is the measure you plan to use. Maybe add it as docstring?